lib/Support/IRHelpers.cpp

```diff
@@ -336,27 +336,28 @@ void dumpUsers(llvm::Value *V) {
 }
 
 static RecursiveCoroutine<void>
-findJumpTarget(llvm::BasicBlock *&Result,
-               llvm::BasicBlock *BB,
-               std::set<BasicBlock *> &Visited) {
+findJumpTarget(const llvm::BasicBlock *&Result,
+               const llvm::BasicBlock *BB,
+               std::set<const BasicBlock *> &Visited) {
   Visited.insert(BB);
 
   if (isJumpTarget(BB)) {
     revng_assert(Result == nullptr,
                  "This block leads to multiple jump targets");
     Result = BB;
   } else {
-    for (BasicBlock *Predecessor : predecessors(BB))
+    for (const BasicBlock *Predecessor : predecessors(BB)) {
       if (!Visited.contains(Predecessor))
         rc_recur findJumpTarget(Result, Predecessor, Visited);
+    }
   }
 
   rc_return;
 }
 
-llvm::BasicBlock *getJumpTargetBlock(llvm::BasicBlock *BB) {
```

Comment:
I described only `getJumpTarget` and not other things, because `getJumpTarget` depends on others, so I had to add `const` in these places to make it compatible. I'll add this to description in commit message.

Reply:
👍🏿